packages/desktop-client/src/components/settings/Upcoming.tsx

```diff
@@ -74,10 +81,8 @@ export function UpcomingLengthSettings() {
         variant="primary"
         onPress={() => setExpanded(true)}
       >
-        <Trans>
-          Edit Upcoming Length (
-          {options.find(x => x.value === upcomingLength)?.label ?? '1 Week'})
-        </Trans>
+        <Trans>Edit Upcoming Length</Trans> (
+        {options.find(x => x.value === upcomingLength)?.label ?? '1 Week'})
```

Comment:
nit: we should still translate '1 week' even if we're removing the extraneous punctuation

Reply:
Done, and added translations for the length options too